crates/db_schema/src/source/moderator.rs

```diff
@@ -219,3 +220,17 @@ pub struct ModAddForm {
   pub other_person_id: PersonId,
   pub removed: Option<bool>,
 }
+
+#[derive(Clone, Queryable, Identifiable, PartialEq, Debug, Serialize)]
+#[table_name = "admin_purge"]
+pub struct AdminPurge {
+  pub id: i32,
+  pub admin_person_id: PersonId,
+  pub when_: chrono::NaiveDateTime,
+}
```

Comment:
Are you planning to integrate this with the mod log, to include a reason etc?

Reply:
I wasn't sure.... since this should only really be used for illegal or really terrible things, its probably not a good idea to include a reason. This would just show `Mod purged an item`. Obvi we can't link to the item purged because its gone from the DB and we don't want to show it.

There's also the question of, should it say the type of item that was purged? If so, I think a string column would be easiest.

With that we could show `Mod purged a comment`

Reply:
You mean that message would be shown in the mod log? Maybe it should also mention the community (and post, if its a comment). Plus it could still show the url to other admins and community mods. And there could be an optional reason.

Reply:
I could do that, add optional community_id, post_id, and reason.